Corona-Warn-App/src/main/res/values-ro/strings.xml

```diff
@@ -574,7 +568,7 @@
     <!-- XACT: settings(tracing) - describes illustration -->
     <string name="settings_tracing_bluetooth_illustration_description_inactive">"O persoană și-a oprit Bluetooth-ul pe smartphone și, prin urmare, întâlnirea cu alte două persoane nu este înregistrată."</string>
     <!-- XACT: settings(tracing) - describes illustration -->
-    <string name="settings_tracing_location_illustration_description_inactive"/>
+    <string name="settings_tracing_location_illustration_description_inactive" />
```

Comment:
please don't change strings files of other languages due to the translation process